packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -118,8 +119,28 @@ export type SdkType =
 export interface SdkBuiltInType extends SdkTypeBase {
   kind: SdkBuiltInKinds;
   encode: string;
+  name: string;
```

Comment:
Could you update https://github.com/Azure/typespec-azure/blob/main/packages/typespec-client-generator-core/doc/types.tsp to make the doc consistent?

Reply:
it has not been updated for a long time, and I have asked Chenjie, it might have been abandoned.
Therefore I will not update it otherwise I will have to catch up quite a few other changes.

Reply:
current doc is: https://azure.github.io/typespec-azure/docs/howtos/DataPlane%20Generation%20-%20DPG/07tcgcTypes
could you help to update this and delete the types.tsp?

Reply:
done